### PR TITLE
Feat(renderer): support handlebar renderer engine

### PR DESCRIPTION
**Pull request by Sorikairox**
Using `@Render` decorator on a route, we can now render a `.hbs` file !

```handlebars
//hello.hbs

<html>
  <head>
    <meta charset="utf-8" />
    <title>{{ title }}</title>
  </head>
  <body>
    Hello {{ name }}!
  </body>
</html>
```

```ts
@Controller('nice-controller')
class MyController {
	@Render('hello')
	@Get('/')
	renderANiceHTML() {
		return { title: 'the page title', name: 'world' };
	}
}
```

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+import { HandlebarRenderer } from './renderer/handlebar.ts';
 import { DanetRouter } from './router/router.ts';
 import { Constructor } from './utils/constructor.ts';
 
 export class DanetApplication {
 	private app = new Application();
 	private injector = new Injector();
 	private hookExecutor = new HookExecutor(this.injector);
-	public danetRouter = new DanetRouter(this.injector);
+	private actualRenderer = new HandlebarRenderer();
```
Comment (rgoupil) [inline src/app.ts:20]: FYI `actual` means "the real/true one", you might be looking for `currentRenderer` or maybe just `renderer`.

**Review by rgoupil — CHANGES_REQUESTED** [review]
Quickly but well done, I only had a minor issue with some variable naming.
Please let me know your opinion about it!

File: src/app.ts (hunk 1 above)
Comment (Sorikairox) [reply src/app.ts:20]: +1

**Review by rgoupil — APPROVED** [review]
LGTM

**Pull request merged**